ash-window/src/lib.rs

```diff
@@ -171,3 +171,69 @@ pub fn enumerate_required_extensions(
 
     Ok(extensions)
 }
+
+/// Query whether a `queue_family` of the given `physical_device` supports presenting to any surface that might be created.
+/// This function can be used to find a suitable [`vk::PhysicalDevice`] and queue family
+/// for rendering before a single surface is created.
```

Comment:
Nit: Can you reflow these sentences to both be approx the same length (line length is 100 characters): https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#max_width